packages/drawnix/src/plugins/with-common.tsx

```diff
@@ -32,5 +34,24 @@ export const withCommonPlugin = (board: PlaitBoard) => {
     return ref;
   };
 
+  const { t } = I18nInsideHook();
```

Comment:
Moving this line to the inside of getI18nValue?
I18nInsideHook is used as normal function, it may have missed the dynamic feature in React context.
So after user change language we may get a uncorrect language.

Reply:
It seems to be good, I need to close this conversion.

Reply:
> I think it's fine that moving this line to the inside of getI18nValue, but I don't think there's difference at placing this inside or outside? Should I put it in?

You are right, keep it.